config/sudoers

```diff
@@ -19,8 +19,9 @@ Defaults	secure_path="/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/b
 # User privilege specification
 root	ALL=(ALL:ALL) ALL
 
-# Allow vx-admin to set hostname and time
+# fine-grained sudo permissions for certain users & actions
 vx-admin ALL=(root:ALL) NOPASSWD: /usr/bin/hostnamectl
 vx-admin ALL=(root:ALL) NOPASSWD: /usr/bin/timedatectl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/bin/timedatectl
 vx-ui ALL=(root:ALL) NOPASSWD: /usr/bin/brightnessctl
+vx-ui ALL=(root:ALL) NOPASSWD: /usr/local/bin/tpm2-totp
```

Comment:
Are there any sub-commands with this that would be bad to allow?

Reply:
I don't see anything that would be a security hole, but you can reset the secret, which would then make all codes not match. I'm not too worried about it for now... unless there's a clear way to limit the parameters? Do we have to wrap it in a separate executable?